src/main/java/org/jabref/logic/util/CoarseChangeFilter.java

```diff
@@ -53,19 +42,30 @@ public synchronized void listen(BibDatabaseContextChangedEvent event) {
             // If editing is started
             boolean isNewEdit = lastFieldChanged.isEmpty();
             // If other field is edited
-            boolean isEditOnOtherField = !isNewEdit && !lastFieldChanged.get().equals(fieldChange.getField());
+            boolean isEditChanged = !isNewEdit && !lastFieldChanged.get().equals(fieldChange.getField());
             // Only deltas of 1 registered by fieldChange, major change means editing much content
-            boolean isMajorChange = totalDelta >= 100;
-
-            if ((isEditOnOtherField && !isNewEdit) || isMajorChange) {
-                // Submit old changes immediately
-                eventPost.run();
-            } else {
-                delayPost.schedule(eventPost);
-            }
+            boolean isMajorChange = totalDelta >= 30;
```

Comment:
Can you state why the magic number 30 has been chosen?

(Java by Comparison: "Replace Comments with Constants")

Reply:
My idea was to collect changes when I manually typing in character by character. Then you only have `delta = 1` for each event. Now I restored it to 1 (as it was before #6663): `delta` is greater than 1 if you paste or delete something. That was my wrong interpretation of a major change happening when `delta > 1`...